### DISCUSSION TITLE
Add Debian 11 (bullseye) to supported supervisor systems.

### DISCUSSION THREAD

**Discussion by Antoni-Czaplicki** [home-assistant/architecture#611] [Entity Models] [answered]
As Debian 11 (bullseye) has been released, I suggest adding it to officially supported supervised installation systems before HA 2021.09 release.

**Comment by frenck** [comment, marked as answer]
This is not a support forum or feature request forum.

This all has been decided and documented already in the ADR.
<https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md>

So I'm not sure what your architectural discussion is about?

**Reply by Antoni-Czaplicki** [reply]
Ah, I only noticed yesterday that you created a pull request in the supervisor repository. This discussion was made to propose this change without creating a PR, as I wasn't sure if this change would be accepted.

**Reply by frenck** [reply]
But, the ADR already covers it, so no need architectural discuss this.
